uart_com.h

```diff
@@ -70,6 +71,12 @@ class uart_com : SensorReader
   int switch_left;
   int switch_right;
   int switch_center;
+  int general;
+  int noise;
+  int cal_act;
+  int i2c_err;
+  int uart_err;
+  bool should_log;
```

Comment:
Please initialize this somewhere. Since it has no initial value, there is a possibility that an incorrect log will be output at the beginning.

Reply:
I modified it so that initialization occurs before calling [uart.start](https://github.com/CMU-cabot/cabot-arduino-ace/blob/95792f199b822d2c26361f5708d11f9e4253f5a9/cabot-arduino-ace.ino#L193C1-L196C18) in the initialization of the main sketch.